_pytest/config.py

```diff
@@ -383,8 +383,8 @@ def import_plugin(self, modname):
             importspec = modname
         try:
             __import__(importspec)
-        except ImportError:
-            raise
+        except ImportError as e:
+            raise ImportError('Error importing plugin "%s": %s' % (modname, e))
```

Comment:
Python >= 3.3 has `name` and `path` attributes for `ImportError` - I think for backwards compatibility, those should be copied over. Probably something like this would work:

```python
new_exc = ImportError('Error importing plugin "%s": %s' % (modname, e))
for attr in ('name', 'path'):
    if hasattr(e, attr):
        setattr(new_exc, getattr(e, attr))
raise new_exc
```